package.json

```diff
@@ -1,27 +1,28 @@
 {
   "name": "bulib-wc",
-  "version": "0.0.12",
+  "version": "0.0.16",
   "description": "collection of web components and styles used at Boston University Libraries",
   "main": "src/index.js",
   "module": "src/index.js",
   "files": [
-    "src/*/*.js",
-    "src/*/*.css",
+    "assets/",
     "dist/bundle*.js",
-    "assets"
+    "dist/bundle.css"
   ],
   "scripts": {
     "clean": "rm -rf node_modules/ || true",
-    "cy:verify": "cypress verify",
     "reinstall": "npm run clean && npm install",
-    "prestart": "npm run build:css",
+    "cy:verify": "cypress verify",
+    "css": "./scripts/bundle_css.sh",
+    "prestart": "npm run css",
     "start": "start-storybook -p 9001",
-    "build": "rollup -c rollup.config.js",
-    "build:css": "scss ./src/header/header.scss > ./src/header/header.css && scss ./src/header/mobile-nav.scss >> ./src/header/header.css",
+    "build:all": "./scripts/rebuild.sh",
+    "build:rollup": "rollup -c rollup.config.js",
     "build:owc": "rollup -c rollup.config.owc.js",
-    "build:storybook": "npm run reinstall && build-storybook -o _site -s storybook-static",
-    "deploy:storybook": "npm run build:storybook && storybook-to-ghpages -o dist",
-    "publish": "npm publish && npm run deploy:storybook"
+    "prebuild:storybook": "npm run css",
+    "build:storybook": "build-storybook -o _site -s storybook-static",
+    "deploy": "./scripts/deploy.sh",
+    "deploy:storybook": "npm run build:storybook && storybook-to-ghpages -o dist"
```

Comment:
this will lead to double-running `build:storybook` and even `npm run css` when running `npm run build:all`, but it's a small price to pay since there is a use case for running it separately.

Reply:
🆗